### DISCUSSION TITLE
Resources not Downloaded

### DISCUSSION THREAD

**Discussion by Alaa-Aim** [vortigont/espem#42] [General]
@vortigont , hope you have a lovely day, I revert back to windows 10, i have an issue when i run respack.sh file, it stocks on "preparing resources from ESPEM image into.../data/ dir", and nothing happen, i tried to run bash as administrator, nothing changed, any advise?

![image](https://github.com/vortigont/espem/assets/64470118/e78ec58b-37a9-485b-b9d2-17429f6351b7)

**Comment by Alaa-Aim** [comment]
I tried WSL, with same outcome, not working 
"D:\espem\resources>bash respack.sh
Preparing resources for ESPEM FS image into ../data/ dir
EmbUI resources requires updating
Refreshing EmbUI css files/pics...
respack.sh: line 113: unzip: command not found
refreshing embui js files...
gzip: embui.js.gz: Operation not permitted
gzip: embui.js.gz: Operation not permitted
gzip: html/index.html.gz: Operation not permitted
gzip: html/index.html.gz: Operation not permitted
gzip: html/js/espem.js.gz: Operation not permitted
gzip: html/js/espem.js.gz: Operation not permitted
gzip: html/js/espem.ui.json.gz: Operation not permitted
gzip: html/js/espem.ui.json.gz: Operation not permitted
gzip: html/css/espem.css.gz: Operation not permitted
gzip: html/css/espem.css.gz: Operation not permitted
gzip: html/manifest.json.gz: Operation not permitted
gzip: html/manifest.json.gz: Operation not permitted
"

**Comment by vortigont** [comment]
Hi, @Alaa-Aim!
Actually you do not need it any more. I decided to include all required files into this repo, so to make a life easier.
Just upload the content of `/data` to your controller and that's it.
Need to update the docs, thanks!

**Comment by Alaa-Aim** [comment]
well noted with many thanks

> Hi, @Alaa-Aim! Actually you do not need it any more. I decided to include all required files into this repo, so to make a life easier. Just upload the content of `/data` to your controller and that's it. Need to update the docs, thanks!
